package.json

```diff
@@ -14,6 +14,7 @@
     "test-watch": "cd packages/tests && pnpm test-watch",
     "lint": "eslint --ext \".js,.ts,.tsx\" --ignore-path .gitignore .",
     "lint-fix": "pnpm lint --fix && manypkg fix",
+    "lint-prune": "! ts-prune | grep -v \"used in module\"",
```

Comment:
Once https://github.com/nadeesha/ts-prune/pull/202 lands, this can be simplified.